sycl/plugins/hip/pi_hip.cpp

```diff
@@ -980,7 +980,15 @@ pi_result hip_piDeviceGetInfo(pi_device device, pi_device_info param_name,
                    PI_DEVICE_TYPE_GPU);
   }
   case PI_DEVICE_INFO_VENDOR_ID: {
-    return getInfo(param_value_size, param_value, param_value_size_ret, 4318u);
+    #if defined(__HIP_PLATFORM_AMD__)
+      pi_uint32 vendor_id = 4098u;
+    #elif defined(__HIP_PLATFORM_NVIDIA__)
+      pi_uint32 vendor_id = 4318u;
+    #else
+      pi_uint32 vendor_id = 0u;
+    #endif
```

Comment:
The indentation is wrong, this needs to be clang-formatted:
```suggestion
#if defined(__HIP_PLATFORM_AMD__)
    pi_uint32 vendor_id = 4098u;
#elif defined(__HIP_PLATFORM_NVIDIA__)
    pi_uint32 vendor_id = 4318u;
#else
    pi_uint32 vendor_id = 0u;
#endif

```